english/justice-league/justice-league.md

```diff
@@ -9,7 +9,7 @@ You send out a distress signal.
 Who will answer the call?
 
   - Superman?
-  - Batman?
```

Comment:
Added a link to a new batman section. Udacity, if you have time please make this merge. Thanks!